stardis/opacities/base.py

```diff
@@ -194,7 +194,7 @@ def calc_tau_line(splasma, marcs_model_fv, tracing_nus):
 
     tau_line = np.zeros([len(marcs_model_fv), len(tracing_nus)])
 
-    gauss_prefactor = 1 / (0.35e10 * np.sqrt(2 * np.pi))
+    gauss_prefactor = 1 / (3.5e10 * np.sqrt(2 * np.pi))
```

Comment:
What is the source of this value?

Reply:
Well I thought it was a mistake, but it turns out it is just a parameter that makes the spectrum look good. This is going to be replaced in the changes in #12 and the future PRs based on it. I still think it's good to merge it to:
- Keep the changes I made in the notebook.
- Avoid merge conflicts with future work.
- Note the results that happen when the parameters change in this way as a reference point.

Thoughts?